packages/utilities/psammead-styles/src/font-styles.test.js

```diff
@@ -0,0 +1,76 @@
+import {
+  getSansRegular,
+  getSansItalic,
+  getSansBold,
+  getSansBoldItalic,
+  getSerifMedium,
+  getSerifMediumItalic,
+} from './font-styles';
+
+const getHelmet = (weight, style) => `
+    font-family: Helmet, Freesans, Helvetica, Arial, sans serif;
+    font-weight: ${weight};
+    font-style: ${style};
+  `;
+
+const getReithSerif = (weight, style) => `
+   font-family: ReithSerif, Helvetica, Arial, sans-serif; 
+   font-style: ${style};
+   font-weight: ${weight};
+  `;
+
+describe('fonts', () => {
+  describe('getSansRegular', () => {
+    it('should return regular sans font css', () => {
```

Comment:
This file seems to only test some of the exports, is it possible to maybe make this a bit easier to read, maybe use config to auto generate the tests? I want to ensure full coverage, but as we get more and more services, verbose tests like this could become weildy. Could we maybe pass a looping test config?

Reply:
Which export are not being tested? This file is supposed to test the exported functions which are all being tested. By all exports do you mean the values of the services?

Reply:
e.g. getSansRegular on yoruba, news, pidgin etc

Reply:
But this would be testing the values of the services not the functions. How do you feel about having a separate test file for `font-families`. And test that the services have the correct values. Because given the correct font values for the services, the functions would work as expected.